이석재/trip_domain_api.yaml

```
@@ -0,0 +1,569 @@
+openapi: 3.0.3
+info:
+  title: Compass Trip System API
+  description: |
+    # Trip 도메인 OpenAPI 명세서
+
+    ## 📋 도메인 구성 요약
+    
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)